Sources/Sentry/Public/SentryDefines.h

```diff
@@ -22,6 +22,9 @@
     -(instancetype)init NS_UNAVAILABLE;                                                            \
     +(instancetype) new NS_UNAVAILABLE;
 
+#define SENTRY_TRUE_STRING @"true"
+#define SENTRY_FALSE_STRING @"false"
```

Comment:
@armcknight Do you think a `#define` is fine or a `const` would be better?

I will create another PR later to replace every instance of "true" and "false"

Reply:
I prefer `static NSString *const` for these over macros, but it's not a huge deal.